sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingQueryListenerSuite.scala

```diff
@@ -81,7 +81,7 @@ class StreamingQueryListenerSuite extends StreamTest with BeforeAndAfter {
       AssertOnLastQueryStatus { status: StreamingQueryStatus =>
         // Check the correctness of the trigger info of the last completed batch reported by
         // onQueryProgress
-        assert(status.triggerDetails.get("triggerId") == "0")
+        assert(Seq("-1", "0").contains(status.triggerDetails.get("triggerId")))
```

Comment:
Look at my PR, I replaced `.get` with `.containsKey` because all we need to test here is that the triggerId key is set. The exact trigger id in which the data was found, etc. is not important. 

https://github.com/apache/spark/pull/15492

Could you change this PR to do the same.